swagger.yaml

```diff
@@ -1121,3 +1121,38 @@ definitions:
       "@type":
         type: string
         format: uri
+  # note: wanted to have a Sinopia: section for Sinopia-not-LDP models, but swagger-codegen just made a SinopiaModels
+  # file and mashed together everything under it, instead of making a file for each model.  so, seems like we need to
+  # do our namespacing via naming convention for one level, instead of hierarchy.
```

Comment:
maybe there can be more model consolidation later (hopefully, if it's appropriate).  for now, this was the easiest way to get the auto-generated code to handle the parameters i'd expect based on the fixtures, in a clean/concise/readable way.  these models allow us to use the SDK constructors to readably build objects to send to trellis/sinopia server.